pipeline/lib/pipeline.ml

```diff
@@ -181,6 +181,7 @@ let github_repositories ?slack_path repo =
        Github.Api.refs api repo
   in
   let default_branch = Github.Api.default_ref refs in
+  let current_timestamp = Util.current_timestamp () in
```

Comment:
The name `current_timestamp` is misleading here. May be something like `stale_commit_timestamp` would work better?